### PR TITLE
Add Backplane dependencies for bp-cli 0.1.7

### PR DESCRIPTION
**Pull request by samanthajayasinghe**
Adding public backplane podman dependencies

### CODE REVIEW

**Comment by T0MASD** [comment]
Please update the PR with the jira reference, if there's no jira please create one, this is very good change that deserves visibility

File: utils/containers.conf [hunk 1]
```diff
@@ -0,0 +1,12 @@
+[containers]
```
Comment (rendhalver) [inline utils/containers.conf:1]: It seems better to put this file into `utils/dockerfile_assets`

Comment (samanthajayasinghe) [reply utils/containers.conf]: Move the file

**Comment by samanthajayasinghe** [comment]
> Please update the PR with the jira reference, if there's no jira please create one, this is very good change that deserves visibility

Thanks, @T0MASD , Created a new JIRA ticket and add the reference

**Comment by iamkirkbater** [comment]
/approve

I think there are still some changes that are requested, but I'll approve the direction of this PR and someone else can LGTM this when it's ready

**Pull request merged**